index.html

```diff
@@ -78,7 +72,12 @@
       // authors, add as many as you like. This is optional, uncomment
       // if you have authors as well as editors. only "name" is
       // required. Same format as editors.
-      authors: [],
+      authors: [{
```

Comment:
```suggestion
      authors: [],
      formerEditors: [{
```

Reply:
I'm fine with this suggestion, I leave it to the current editors to apply it.